src/test/java/org/broadinstitute/hellbender/tools/TestSpecUnitTest.java

```diff
@@ -0,0 +1,23 @@
+package org.broadinstitute.hellbender.tools;
+
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * Testing the test infrastructure
+ */
+public final class TestSpecUnitTest {
```

Comment:
Could you add a test case with a file that has a comment, and the same file without the comment, and show that it is not equal when you don't ignore the comments but it is equal when you do?